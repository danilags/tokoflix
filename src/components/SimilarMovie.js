import React from 'react';
import {
  Card,
  CardBody,
  CardText,
  Media
} from 'reactstrap';

import { IMAGE_BASE_URL } from '../constants';

const SimilarMovie = ({ item }) => (
  <Card>
    <CardBody>
      <Media className="mt-1">
        <Media heading>
          <img style={{ width: '100%' }} src={`${IMAGE_BASE_URL}${item.backdrop_path}`} alt={item.title} />
        </Media>
      </Media>
      <CardText>
        <h3 style={{ color: '#4a4a4a' }}>{item.title}</h3>
      </CardText>
      <p>{item.overview}</p>
    </CardBody>
  </Card>
);

export default SimilarMovie;
