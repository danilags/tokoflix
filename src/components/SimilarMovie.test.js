/**
 * Testing SimilarMovie component
 */

import React from 'react';
import { mount } from 'enzyme';
import renderer from 'react-test-renderer';

import SimilarMovie from './SimilarMovie';

describe('<SimilarMovie />', () => {
  // it('Component renders and matches snapShot', () => {
  //   let data = {
  //     backdrop_path: 'http://anotherurl.com/7eTWQb475pM9H3NEOiUzGYASEx0.jpg',
  //     title: 'Jurasic World',
  //     overview: 'Lorem Ipsum Dolor Sit amet'
  //   }
  //   const SimilarMovie = shallow(<SimilarMovie item={data} />)
  //   expect(SimilarMovie).toMatchSnapshot();
  // });

  it('test SimilarMovie matches MOUNT', () => {
    let inst = renderer.create(React.createElement('div', <SimilarMovie />));
    expect(inst.toJSON()).toMatchSnapshot();
  });
})


