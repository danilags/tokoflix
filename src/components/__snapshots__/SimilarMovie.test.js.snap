// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SimilarMovie /> test SimilarMovie matches MOUNT 1`] = `
<div
  $$typeof={Symbol(react.element)}
  _owner={null}
  _store={Object {}}
  props={Object {}}
  type={[Function]}
/>
`;
